ios/Classes/MapboxMapController.swift

```diff
@@ -207,6 +207,9 @@ class MapboxMapController: NSObject, FlutterPlatformView, MGLMapViewDelegate, Ma
                 style.localizeLabels(into: locale)
             }
             result(nil)
+        case "map#setMaximumFps":
+            // not implemented, no support for this on iOS
+            result(nil)            
```

Comment:
might be better to return an error here - otherwise the user might wonder why it doesn't work. If we do it like so it's would be up to the user to either catch the error only set this on android.

What do you think?